src/mixs/schema/mixs.yaml

```diff
@@ -16961,8 +16997,18 @@ classes:
         examples:
         - value: 1.6 kilometers per hour
     class_uri: MIXS:0016020
+    aliases:
+      - MIxS-Food (farm environment)
+    annotations:
+      use_cases: Microbiome of farm and field crops as well as environmental samples including irrigation, soil amendments, and farm equipment.
   FoodFoodProductionFacility:
```

Comment:
I think we can make a separate issue for this.. but is it really foodfood? Why is food twice?

Reply:
Good question. That is not a typo or bug. In this case, it is helpful to look at the submitter-provided `title` of this class: 'food-food production facility'. Pre-LinkML, an attempt was made to **approximate** hierarchy in the `Extension`s, as if there is a category of "food" `Extension`s, of which 'food production facility' is a child `Extension`. 

Following [standard naming for LinkML classes](https://linkml.io/linkml/schemas/linter.html#standard-naming), the `title` `food-food production facility` has been transformed into a term `name` of 'FoodFoodProductionFacility'.

I don't have any objection t changing that, but it should be done systematically.

Reply:
@turbomam is correct, it was an attempt at hierarachy, the FDA were involved in designing a set of 4 extensions, so they grouped them all under a "Food" heading. 

- MIxS-food-animal and animal feed
- MIxS-food-farm environment
- MIxS-food-food production facility
- MIxS-food-human foods

I'm not against renaming them, but there maybe parties that might be opposed to it, so I have opened a discussion ticket for it #779